superset-frontend/src/components/Chart/types.ts

```diff
@@ -17,6 +17,11 @@
  * under the License.
  */
 
+export enum DrillByType {
+  chart,
```

Comment:
We mix every imaginable naming convention for enums in Superset, but I think the one we agreed to use some time ago is PascalCase 😄